.github/workflows/build.yml

```diff
@@ -16,6 +16,10 @@ jobs:
         destination: ['platform=iOS Simulator,OS=14.0,name=iPhone 11 Pro']
 
```

Comment:
@rajdeep Also, instead of running `xcode-select` in steps you can set DEVELOPER_DIR environment variable before:
```
    env: 
      DEVELOPER_DIR: /Applications/Xcode_12.2.app/Contents/Developer
```
Not much difference though, just another option.

Reply:
Yup! I saw that option too. Just felt like going with this. I'll anyways be looking at why it failed on 12.4 so this is going to be interim fix only.